src/components/SearchForm/style.css

```diff
@@ -0,0 +1,47 @@
+@import url('https://fonts.googleapis.com/css2?family=Nunito:wght@600;800&display=swap');
+#form{
+  font-weight: bolder;
+  font-size: 1.2rem;
+  font-family: 'Nunito', sans-serif;
+  padding-right: 5px;
+}
+
+#form-container {
+  text-align: center;
+}
+
+input [type=text] {
+  width: 100%;
+  font-size: 1.5rem;
+  font-weight: bolder;
+  padding: 12px 20px;
+  margin: 8px 0px;
+  border: 2px solid #555;
+  transition: width 0.4s ease-in-out;
+}
+
+input[type=text]:focus {
+    background-color: lightblue;
+    width: 20rem;
+    height: 2rem;
+    border: 4px solid black;
+}
```

Comment:
when the user goes to click submit button the input shrinks in size as it loses focus, which shifts the button away from where the user is clicking, and this seems to fire before the click on the button registers -  I think just make sure the submit button is still in the location it was when user clicked !